src/cmds/scripts/pbs.service.in

```diff
@@ -55,7 +55,7 @@ DefaultDependencies=true
 [Service]
 Type=forking
 Restart=no
-TimeoutSec=5min
+TimeoutSec=0
```

Comment:
Systemd terminates the start up process on reaching timeout. Setting it to infinity should not be a problem, at least not as far as I can tell.